src/components/BorrowedBook.tsx

```diff
@@ -4,17 +4,22 @@ import { getBookImage } from '../utils/utils';
 import Rating from './Rating';
 import './Book.css';
 import 'react-toastify/dist/ReactToastify.css';
+import ReturnBook from './ReturnBook';
 
 interface IBookProps {
+  id: string | undefined;
```

Comment:
I think every book in our db has to have an ID
```suggestion
  id: string;
```